src/Extensions/AbstractWebDriverExtension.php

```diff
@@ -17,33 +17,42 @@
 use Codeception\Exception\ModuleRequireException;
 use Codeception\Module\WebDriver;
 
+/**
+ * Base class for extensions
+ */
 abstract class AbstractWebDriverExtension extends AbstractCodeceptionExtension
 {
     /**
      * @var WebDriver
      */
-    protected $webDriverModule;
+    protected \Codeception\Module\WebDriver $webDriverModule;
 
     /**
      * @var string
      */
-    protected $environment;
+    protected string $environment = 'testing';
 
+    /**
+     * @param SuiteEvent $event
+     * @return void
+     */
     public function loadWebDriver(SuiteEvent $event): void
     {
         try {
-            $this->webDriverModule = $this->webDriverModule = $this->getModule('WebDriver');
+            $this->webDriverModule = $this->getModule('WebDriver');
         } catch (ModuleRequireException $moduleRequireException) {
             throw new ExtensionException($this, 'This extension requires the WebDriver module!');
         }
     }
 
+    /**
+     * @param SuiteEvent $event
+     * @return void
+     */
```

Comment:
Can you remove the superfluous @param and @return in the docblock they are already specified in the code. Thanks